e2e/starter.test.js

```diff
@@ -0,0 +1,23 @@
+describe('Example', () => {
```

Comment:
Sample failing tests to check Detox setup.

Reply:
Provided by Detox docs.

Reply:
https://wix.github.io/Detox/docs/introduction/your-first-test#5-make-an-assertion